salt/returners/slack_returner.py

```diff
@@ -14,6 +14,7 @@
     slack.as_user (required to see the profile picture of your bot)
     slack.profile (optional)
     slack.changes(optional, only show changes and failed states)
+    slack.failed(optional, only show failed states)
```

Comment:
Would this be more clear if the option was `only_show_failed`?

Reply:
That is more clear as to what the option does and I have implemented that suggestion.